build.sbt

```diff
@@ -281,7 +281,7 @@ lazy val commonSettings =
       )
     )
 
-val flinkV = "1.16.1"
+val flinkV = "1.16.2"
```

Comment:
In tests we also have Dockerfile which is based on Flink docker image (1.16.1)

Reply:
Thanks, fixed